rcl/test/rcl/service_fixture.cpp

```diff
@@ -149,6 +149,9 @@ int main(int argc, char ** argv)
       return -1;
     }
     // Our scope exits should take care of fini for everything
+    // stick around until we are killed by the client
+    // To avoid an infinite loop and signal handling, just sleep for a "long" time
+    std::this_thread::sleep_for(std::chrono::seconds(3));
```

Comment:
After discussing with @jacquelinekay offline, we agreed that adding a new exit handler to launch which allows for the "sigterm" return code after the server was interrupted by launch is the right way to go. Then the server can just busy-loop wait forever rather than do a single sleep here.